README.md

```diff
@@ -4,9 +4,15 @@ Add sane drag and drop and copy/paste support for images to markdown
 files. This is especially useful for notetaking in Atom.
 
 Whenever an image is dragged and dropped or copy and pasted on an open
-markdown file, this package automatically copies it to an `assets/` folder
+markdown file, this package automatically copies it to the folder `<fileroot>.assets/`
 in the current directory, renames the image, and inserts a relative link
-to the image.
+to the image.   
+
+`<fileroot>` is the root of the file without the supported Markdown extensions (`.markdown`, `.md`, `.mdown`, `.mkd`, `.mkdow`).
+
+![](README.assets/README-54317719.png)
+
+-----
 
```

Comment:
This description makes this look like the default. Maybe add a section called options and add this description under there? 

Also, I would be careful about replacing `assets/` with `<fileroot>.assets/`. The former does not have the dot.